lib/util/init.config.js

```diff
@@ -179,7 +184,11 @@ const Init = {
         } else {
             const fileContent = File.readFileSync(gitignoreFileName, 'utf8');
             creationLog.push(
-                await this._createIdeConfigFile(['.' + path.sep, '', '.gitignore'], fileContent)
+                await this._createIdeConfigFile(
+                    ['.' + path.sep, '', '.gitignore'],
```

Comment:
please use path.join or path.resolve here

Reply:
we do that later inside of _createIdeConfigFile - after that function adds some more to the array. running one of the path methods here would make things more complicated and essentially require it to run twice

Reply:
that and we kinda need that array down the road:
![image](https://user-images.githubusercontent.com/1917227/164087327-2540d09a-32cc-4f8b-9404-adb31cbc9be6.png)

that's why the array is explained in _createIdeConfigFile's jsdoc:
![image](https://user-images.githubusercontent.com/1917227/164087419-e422815b-c8eb-46f5-8070-a7eb4df0bd88.png)